### PR TITLE
test(wkt): more primitive types integration tests

### PR DESCRIPTION
**Pull request by coryan**
Add more integration tests where primitive types appear as fields of
messages.

This made the `message_with_map` test redundant, we test the map fields
with each primitive type.

Motivated by #2349, I will need these tests in place when the "null as default" thing is implemented.

### CODE REVIEW

File: src/wkt/tests/message_with_bool.rs [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        let output = json!({"singular": true});
+        assert_eq!(got, MessageWithBool::new().set_singular(true));
+        let trip = serde_json::to_value(&got)?;
+        assert_eq!(trip, output);
+        Ok(())
+    }
+
+    #[test_case(json!({}))]
+    #[test_case(json!({"singular": false}))]
+    // TODO(#2349) - #[test_cast(json!({"singular": null}))]
```
Comment (dbolduc) [inline src/wkt/tests/message_with_bool.rs:41]: `test_case`

(But dont bother changing it now)

File: src/wkt/tests/message_with_string.rs [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        let output = json!({"singular": input});
+        assert_eq!(got, MessageWithString::new().set_singular(input));
+        let trip = serde_json::to_value(&got)?;
+        assert_eq!(trip, output);
+        Ok(())
+    }
+
+    #[test_case(json!({}))]
+    #[test_case(json!({"singular": ""}))]
+    // TODO(#2349) - #[test_cast(json!({"singular": null}))]
```
Comment (dbolduc) [inline src/wkt/tests/message_with_string.rs:43]: same
